internal/controller/ansibleRun/ansibleRun.go

```diff
@@ -155,16 +159,38 @@ func (c *connector) Connect(ctx context.Context, mg resource.Managed) (managed.E
 	// garbage collector that is started in Setup.
 	dir := filepath.Join(baseWorkingDir, string(cr.GetUID()))
 	if err := c.fs.MkdirAll(dir, 0700); resource.Ignore(os.IsExist, err) != nil {
-		return nil, errors.Wrap(err, errMkdir)
+		return nil, fmt.Errorf("%s: %s: %w", baseWorkingDir, errMkdir, err)
 	}
 
 	if err := c.usage.Track(ctx, mg); err != nil {
-		return nil, errors.Wrap(err, errTrackPCUsage)
+		return nil, fmt.Errorf("%s: %w", errTrackPCUsage, err)
 	}
 
 	pc := &v1alpha1.ProviderConfig{}
 	if err := c.kube.Get(ctx, types.NamespacedName{Name: cr.GetProviderConfigReference().Name}, pc); err != nil {
-		return nil, errors.Wrap(err, errGetPC)
+		return nil, fmt.Errorf("%s: %w", errGetPC, err)
+	}
+
+	// Saved inventory needed for ansible content hosts
+	var buff bytes.Buffer
+	for _, i := range cr.Spec.ForProvider.Inventories {
+		data, err := resource.CommonCredentialExtractor(ctx, i.Source, c.kube, i.CommonCredentialSelectors)
+		if err != nil {
+			return nil, fmt.Errorf("%s: %w", errGetInventory, err)
+		}
+		if _, err := buff.WriteString(string(data) + "\n"); err != nil {
+			return nil, err
+		}
+	}
+	if cr.Spec.ForProvider.InventoryInline != nil {
```

Comment:
What will happen when both InventoryInline and Inventories are defined? Is this allowed?